src/styles/starts.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+  }
+
+  &-container {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+    align-items: center;
+  }
+
+  &-container__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    margin: 2px 0 0;
+    color: #060b35;
+  }
+
+  &--0 {
+    .stars__star:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg); /* Active star image */
+    }
```

Comment:
The BEM modifier `stars--0` seems to be misnamed. According to the checklist, modifiers should represent different states or versions of a block or an element. However, `stars--0` implies it is for zero stars, which doesn't make sense as this code is setting the `star-active` image for the first four stars. Ensure that the modifier name and the associated style rules accurately represent the intended design and functionality.